src/MoBi.Core/Services/PKSimStarter.cs

```diff
@@ -205,13 +205,14 @@ private string retrievePKSimUIStarterPath()
 
       private string retrievePKSimExecutablePath()
       {
-         //Installed properly via Setup? return standard path
-         if (FileHelper.FileExists(_configuration.PKSimPath))
-            return _configuration.PKSimPath;
-
+         // Specified explicitly via user settings? Use overridden path
          if (FileHelper.FileExists(_applicationSettings.PKSimPath))
             return _applicationSettings.PKSimPath;
 
+         // Installed properly via Setup? Use standard path
+         if (FileHelper.FileExists(_configuration.PKSimPath))
+            return _configuration.PKSimPath;
+
```

Comment:
I reversed the order here. This is now checking the user specified location first, then the registry location

Reply:
or was it from PKSim => starting Mobi